### DISCUSSION TITLE
Have timeline available in camera groups

### DISCUSSION THREAD

**Discussion by jchretien** [blakeblackshear/frigate#12773] [Polls]
I like the camera group feature and I think the "Live" view should just be a default camera group with every cameras.

The "motion" tab is just a camera group with the timeline and the camera size can't be adjusted. Why not have that timeline in any camera group? Basically combine every cool features in a super tab.

**Comment by NickM-27** [comment]
There are a number of reasons why combining live and review for example is technically difficult and go against the principles of the new UI. This was specifically decided against this during the design phase (read more about that https://github.com/blakeblackshear/frigate/discussions/11136)

Having a timeline in the live page for example would then bring about the desire to go back further in time which would encourage the need for a date picker. Users also want to filter what is shown on the timeline which adds more UI elements and it can very easily end up in a cluttered UI. Viewing live cameras is a different goal ("what is happening or just happened on my cameras"?) than the goal ("what happened on my cameras last night?) or many other goals that have yet to be implemented.

**Reply by jchretien** [reply]
Then why can't we resize the cameras in the review Tab? When action was going on many cameras at once I'd like to watch them together, not one by one.
![image](https://github.com/user-attachments/assets/93dda28e-a75a-4802-a938-3bb9b5b0d5e3)

**Reply by NickM-27** [reply]
That may be possible in the future, but has not been implemented for the initial version
